com.jvieira.tpt.Metronome.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It seems that you're having an appdata file upstream already, any reason for including it here as well?

Reply:
No reason, I thought it was necessary...

Reply:
It's necessary to be part of the app, Flathub only requires the manifest https://github.com/flathub/flathub/wiki/App-Requirements

Reply:
So, I will make the suggested changes in the upstream appdata file, and I will remove it from here.